mdsf/src/tools/biome_format.rs

```diff
@@ -66,20 +63,23 @@ mod test_biome_format {
     }
 
     #[test_with::executable(npx)]
-    fn test_biome_format_javascript_8a9233d85bad9eb0() {
+    fn test_biome_format_json_90a326e29048e3cd() {
         let input = r#"
-    async function asyncAddition(
-            a,b
-        ) {
-        return a+b
-    }
-
-            "#;
-        let output = r#"async function asyncAddition(a, b) {
-	return a + b;
+              {
+              "key": "value",
+  "key2": [
+      "value2",
+      "value3",
+      1
+            , null]
+ }
+  "#;
+        let output = r#"{
+	"key": "value",
+	"key2": ["value2", "value3", 1, null]
 }
 "#;
-        let file_ext = crate::fttype::get_file_extension("javascript");
+        let file_ext = crate::fttype::get_file_extension("json");
```

Comment:
Here it appears to me autogenerated sections are provided in random order.

Is there anything you can do?

Reply:
They are now ordered by the order they are defined in the `tests` field 

https://github.com/hougesen/mdsf/blob/90ae646726dda94e089eb61b35c13fbba2052b64/tools/biome/plugin.json#L16-L35

Reply:
Marvelous, it would avoid the noise in PR, thanks